content/en/docs/concepts/metrics/_index.md

```diff
@@ -1333,15 +1215,41 @@ falcosecurity_scap_signal_deliver_run_time_nanoseconds_total 0
 # HELP falcosecurity_scap_signal_deliver_avg_time_nanoseconds https://falco.org/docs/metrics/
 # TYPE falcosecurity_scap_signal_deliver_avg_time_nanoseconds gauge
 falcosecurity_scap_signal_deliver_avg_time_nanoseconds 0
+# HELP falcosecurity_falco_kernel_release_info https://falco.org/docs/metrics/
+# TYPE falcosecurity_falco_kernel_release_info gauge
+falcosecurity_falco_kernel_release_info{kernel_release="6.14.6-300.fc42.x86_64"} 1
+# HELP falcosecurity_evt_hostname_info https://falco.org/docs/metrics/
+# TYPE falcosecurity_evt_hostname_info gauge
+falcosecurity_evt_hostname_info{hostname="test"} 1
+# HELP falcosecurity_falco_outputs_queue_num_drops_total https://falco.org/docs/metrics/
+# TYPE falcosecurity_falco_outputs_queue_num_drops_total counter
+falcosecurity_falco_outputs_queue_num_drops_total 0
+# HELP falcosecurity_falco_reload_timestamp_nanoseconds https://falco.org/docs/metrics/
+# TYPE falcosecurity_falco_reload_timestamp_nanoseconds gauge
+falcosecurity_falco_reload_timestamp_nanoseconds 1748559099987386014
+# HELP falcosecurity_falco_start_timestamp_nanoseconds https://falco.org/docs/metrics/
+# TYPE falcosecurity_falco_start_timestamp_nanoseconds gauge
+falcosecurity_falco_start_timestamp_nanoseconds 1748559099986267926
+# HELP falcosecurity_falco_duration_seconds_total https://falco.org/docs/metrics/
+# TYPE falcosecurity_falco_duration_seconds_total counter
+falcosecurity_falco_duration_seconds_total 96
+# HELP falcosecurity_falco_host_boot_timestamp_nanoseconds https://falco.org/docs/metrics/
+# TYPE falcosecurity_falco_host_boot_timestamp_nanoseconds gauge
+falcosecurity_falco_host_boot_timestamp_nanoseconds 1748555738000000000
+# HELP falcosecurity_falco_host_num_cpus_total https://falco.org/docs/metrics/
+# TYPE falcosecurity_falco_host_num_cpus_total gauge
+falcosecurity_falco_host_num_cpus_total 8
 ```
 
 </details>
 
+
 <details>
-  <summary> Show All Fields
```

Comment:
These examples are tedious to maintain -- did a best effort pass of rerunning Falco w/ each metric config enabled individually. I also removed the listing of all fields at the end at this time @leogr.
Likely this page won't need syncing for some time again.